sdk/python/README.md

```diff
@@ -2,10 +2,13 @@
 
 python library to interact with PV Site database
 
+## Structure
+see https://blog.ionelmc.ro/2014/05/25/python-packaging/#the-structure%3E
+
 ## Read functions
 
 - get_site
-- 
```

Comment:
Is it worth adding the read functions here, just so its easy to read

Reply:
Updated README.md with [`394c529` (#25)](https://github.com/openclimatefix/pvsite-datamodel/pull/25/commits/394c5290cb6a0c894d66cfc50486358f384a2f58)